oomact_python/test/tests.py

```diff
@@ -77,17 +77,17 @@ def __init__(self, *args, **kwargs):
     
     def testWheelOdometryCalibrationStraight(self):
         #load config file somehow (does not seem to exist)
-        #vs = sm.ValueStoreRef()
-        #vs=vs.fromFile("acceptance/wheelOdometry-pose.info") #does this not exist for python yet
         vs=sm.valueStoreRefFromFile("/home/yannick/catkin_ws/src/oomact/oomact/test/acceptance/wheelOdometry-pose.info")
 
         #this does not work yet
```

Comment:
Is this comment up to date?

Reply:
No valuestoreref is working, but from file is not a class function, i dont know if that is an issue